features/gezag-minderjarige/1.1-ingezetene-BRP.feature

```diff
@@ -0,0 +1,122 @@
+#language: nl
+
+Functionaliteit: 1.1 - Staat persoon (minderjarige) als ingezetene in de BRP?
+
+
+    Achtergrond:
+      Gegeven de persoon 'P1' met burgerservicenummer '000000012'
+      * is meerderjarig
+      * heeft geslachtsnaam gevuld
```

Comment:
voor mij is het interessanter om te weten hoe er gecommuniceerd moet worden dat iemand de . (punt) als geslachtsnaam heeft.  `heeft geen / de standaard geslachtsnaam`. `heeft geslachtsnaam gevuld` kan standaard worden ingevuld zodat je het niet voor elke scenario dit hoeft aan te geven

Reply:
ik ben het daarmee eens. zal deze stappen verwijderen